lib/widgets/slot_alert_dialog.dart

```diff
@@ -60,8 +80,9 @@ class _SlotAlertDialogState extends State<SlotAlertDialog> {
               borderRadius: BorderRadius.circular(3),
               image: DecorationImage(
                 fit: BoxFit.cover,
-                image: NetworkImage(
-                    'https://firebasestorage.googleapis.com/v0/b/parking-41df9.appspot.com/o/slot.jpg?alt=media&token=1015cf8b-0613-4c0d-b42d-df7341a0ee5f'),
+                image: NetworkImage(widget.slot['imageUrl'] != null
+                    ? widget.slot['imageUrl']
+                    : 'https://firebasestorage.googleapis.com/v0/b/parking-41df9.appspot.com/o/slot.jpg?alt=media&token=1015cf8b-0613-4c0d-b42d-df7341a0ee5f'),
```

Comment:
Tokens shouldn't be in git for safety reasons

Reply:
I didn't notice that, it should be a normal link to an image stored in firebase-storage, but I will check how to solve that problem.